scripts/easy_train.py

```diff
@@ -0,0 +1,1844 @@
+            features = features[:-1]
+
+        with subprocess.Popen(
+            [
+                sys.executable,
+                'serialize.py',
+                os.path.abspath(model_path),
+                destination_model_path,
+                f'--features={features}',
+            ],
+            cwd=nnue_pytorch_directory,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT
+        ) as process:
+            if process.wait():
+                raise Exception('Failed to run serialize.py for start model.')
+
+        if not os.path.exists(destination_model_path):
+            raise Exception('Failed to convert start model.')
+
+    return destination_model_path
+
+def prepare_start_model_from_experiment(directory, experiment_path, run_id, nnue_pytorch_directory, features):
+    root_dir = os.path.join(experiment_path, 'training')
+    best_model = find_best_checkpoint(root_dir)
+    if best_model is None:
+        best_model = find_latest_checkpoint(root_dir)
+    if best_model is None:
+        raise Exception('Could not find any viable .ckpt nor .nnue files in the start experiment.')
+    return prepare_start_model(directory, best_model, run_id, nnue_pytorch_directory, features)
+
+def get_default_feature_set_from_nnue_pytorch(nnue_pytorch_directory):
+    with open(os.path.join(nnue_pytorch_directory, 'features.py'), 'r') as features_file:
+        for line in features_file:
+            line = line.strip()
+            if line.startswith('_default_feature_set_name'):
+                return line.split()[-1][1:-1]
+
+def main():
+    LOGGER.info('Initializing...')
+
+    args = parse_cli_args()
+
+    # if we ask to resume don't fail on existing directory
+    if args.resume_training:
+       args.fail_on_experiment_exists=False
+
+    absolute_workspace_path = os.path.abspath(args.workspace_path)
+    experiment_directory = os.path.join(absolute_workspace_path, f'experiments/experiment_{args.experiment_name}')
+    try:
+        os.makedirs(experiment_directory, exist_ok=False)
+    except FileExistsError as e:
+        if args.fail_on_experiment_exists and os.listdir(experiment_directory):
+            LOGGER.error(f'Directory {experiment_directory} already exists. An experiment must use a new directory.')
+            LOGGER.error(f'Alternatively, override this with the option --resume-training=True or --fail-on-experiment-exists=False.')
+            return
+
+    ordo_directory = os.path.join(absolute_workspace_path, 'ordo')
+    c_chess_cli_directory = os.path.join(absolute_workspace_path, 'c-chess-cli')
+    books_directory = os.path.join(absolute_workspace_path, 'books')
+
+    stockfish_base_directory = os.path.join(experiment_directory, 'stockfish_base')
+    stockfish_test_directory = os.path.join(experiment_directory, 'stockfish_test')
+    nnue_pytorch_directory = os.path.join(experiment_directory, 'nnue-pytorch')
+    bookkeeping_directory = os.path.join(experiment_directory, 'bookkeeping')
```

Comment:
maybe this directory could be 'logging' as that is what we store there right now?